setup.py

```diff
@@ -39,7 +39,7 @@
       keywords="web services",
       author='Mozilla Services',
       author_email='storage@mozilla.com',
-      url='https://kintoclient.readthedocs.org',
+      url='https://kintoclient.readthedocs.io',
```

Comment:
Actually the previous link didn't exists and the new one doesn't either. I would use the github repository address instead.